Userland/Libraries/LibTimeZone/TimeZone.cpp

```diff
@@ -20,15 +22,77 @@ enum class TimeZone : u16 {
 };
 #endif
 
+class TimeZoneFile {
+public:
+    TimeZoneFile(char const* mode)
+        : m_file(fopen("/etc/timezone", mode))
```

Comment:
> Huh? I don't see any references to this file... Don't we use `/etc/timezone`?

Oops! That was a dangling reference to an old impl of this PR, good catch.
 
> I feel like this commit should be squashed into the previous one, otherwise the affected applications may stop working when bisecting.

I'll move that commit to be one earlier - my initial thought was that the commit order didn't matter there, because the LibC method wasn't using /etc/timezone yet, but on closer look that isn't quite right, WebContent / Browser would get tripped up.

Reply:
Right, so tbh I'm not 100% what the right thing to do here is yet. I'm not expecting host systems to have /etc/timezone. But my (limited) understanding of this stuff is that if your program cares about time zones, then it should `tzset` at startup. That method should check the `TZ` var, then fall back to a system-dependent default. My intention is that reading this file is the system-dependent default for Serenity. Then, eventually, the `localtime` impl here will respect what `tzset` set up.

I've held off on implementing that yet because I'm not quite sure how that allows for reacting to time zone changes. This current impl allows for changes, but I'm sure needs to be thought about further.